.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -30,3 +30,63 @@ Describe the grammar changes or new functions.
 
 Why problems is this JEP addressing?
 Has alternatives been considered?
+
+## Implementation Survey
+
+*Omit this section if not relevant*
+
+Provide a minimal demonstration of how the proposed functionality would be implemented for each programming language.
+Include links to any proposed dependencies. This is to ensure that each language can support the proposed functionality, allows for
+the review of any additional dependencies that would need to be included with each implementation, provides reference for implementers,
+and ensures that each proposed implementation in each language has consistent behaviour across all languages.
+
+
+### Javascript/Typescript

```

Comment:
Not sure if we should have them all.
At this rate, this could become unwieldy.

I think Elixir libraries are now available as well ?
Is a Lua library available ; I will need to add this to the list of supported libraries.

Reply:
https://github.com/jmespath/jmespath.lua

I have never heard of Elixir, I'll add it.